projects/vfat/index.js

```diff
@@ -162,52 +87,48 @@ async function fetchGauges2(api, fromBlock, gaugeFactory, gaugeFactory2, voter,
 }
 
 async function fetchSickleNftPositions(api, sickles, managerAddress, isMasterchef = false) {
-  const sickleBalances = {};
-
-  for (const sickle of sickles) {
```

Comment:
I think this line made it slow, we are checking one sickle at a time, I turned it into a batch call @vfat0